src/Components/Common/OnlineDoctorsSelect.tsx

```diff
@@ -38,8 +59,17 @@ export const OnlineDoctorsSelect = (props: any) => {
         </label>
         <div className="relative">
           <span className="inline-block w-full rounded-md shadow-sm">
-            <button onClick={_ => setIsExpanded(!isExpanded)} type="button" aria-haspopup="listbox" aria-expanded="true" aria-labelledby="listbox-label" className="cursor-default relative w-full rounded-md border border-gray-300 bg-white pl-3 pr-10 py-2 text-left focus:outline-none focus:shadow-outline-blue focus:border-blue-300 transition ease-in-out duration-150 sm:text-sm sm:leading-5">
-              <div className="flex items-center justify-between">
+            <button onClick={_ => setIsExpanded(!isExpanded)} type="button" aria-haspopup="listbox" aria-expanded="true" aria-labelledby="listbox-label" className={`cursor-default relative w-full rounded-md border border-gray-300 bg-white pl-3 pr-10 py-2 text-left focus:outline-none focus:shadow-outline-blue focus:border-blue-300 transition ease-in-out duration-150 sm:text-sm sm:leading-5`}>
+              <input
+                ref={searchFieldRef}
+                name="searchTerm"
+                type="text"
+                placeholder="Search by username"
+                className={`py-2 pl-3 w-full outline-none ${!isExpanded ? 'hidden' : ''}`}
```

Comment:
Can we use `classnames` library here? It's already a dependency

Reply:
nice 😃